frontend/src/pages/Traces/TracesPage.tsx

```diff
@@ -271,23 +275,47 @@ export const TracesPage: React.FC = () => {
 								alignItems="center"
 								display="flex"
 								flexDirection="row"
-								px="10"
-								mb="4"
-								gap="10"
-								style={{ height: 28 }}
+								px="8"
+								pt="4"
+								pb="6"
+								gap="8"
 							>
 								{metricsLoading ? (
-									<HistogramLoading />
+									<Box py="4">
+										<HistogramLoading />
+									</Box>
 								) : (
 									<>
-										<Text size="xSmall" color="weak">
+										<Tag
```

Comment:
a bit weird that the card has a hover state that makes it look like a button when it's not clickable

![Screenshot from 2024-06-21 11-16-24](https://github.com/highlight/highlight/assets/1351531/0e3d522d-da1b-4a16-b1b0-83fe5aa5b718)
![image](https://github.com/highlight/highlight/assets/1351531/0055235c-43cd-44aa-bfa2-64441649101b)

Reply:
Yeah, I agree. @julian-highlight I think this is a problem with using a `Tag` for something that is purely visual. Perhaps a `Badge` would make more sense here? I will swap out for a badge, but I think used tag initially because it's what I saw in the mocks.